### PR TITLE
Fix some syntactical errors and improve readability

### PR DESCRIPTION
**Pull request by mcmahonjohn**
* specification_v1_0.html - Remove extra closing </li>
* helpers.js - Remove unused parameter *context* from function checkNodeImport
 * Improve readability in functions checkNodeImport and prepend
* index.js -  Add braces to if statement to improve readability
* angular2-hmr-prod.js - Add missing semicolon
* webpack.common.js - Remove extra commas (',')
* webpack.dev.js - Remove extra commas (',')
* webpack.github-deploy.js - Remove unused parameter *stats* from function
* webpack.prod.js - Remove extra commas (',')
 * Remove redundant character escape from *customAttrAssign*
* webpack.test.js - Remove extra commas (',')

### CODE REVIEW

File: config/webpack.github-deploy.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ module.exports = webpackMerge(webpackConfig, {
 
   plugins: [
     function() {
-      this.plugin("done", function(stats) {
+      this.plugin("done", function() {
```
Comment (mcmahonjohn) [inline config/webpack.github-deploy.js]: Good catch Mr. Bot, I'll address that.

**Comment by mcmahonjohn** [comment]
This is interested, CodeClimate doesn't find any issues, but CircleCi fails because when it runs codeclimate-test-reporter it does not find an environment variable for CODECLIMATE_REPO_TOKEN.

Here's some documentation for [how to set environment variables](https://circleci.com/docs/environment-variables/#setting-environment-variables-for-all-commands-without-adding-them-to-git) in your project settings on CircleCI. I don't have the access to address this unsuccessful check.

**Comment by lukad03** [comment]
@mcmahonjohn Thanks for the PR's!

We have the CodeClimate Repo Token defined in the project's settings in CircleCI. By default, CircleCI [doesn't allow these variables to be accessed by PRs from forks](https://circleci.com/docs/fork-pr-builds/) because they could be exposed. I've enabled builds to be generated from forks in CircleCI so we're all set.

**Comment by mcmahonjohn** [comment]
Cool, thanks @lukad03 !

**Pull request merged**